cmd/rhobs/alerts_cmd.go

```
@@ -0,0 +1,647 @@
+			}
+
+			return nil
+		},
+	}
+	cmd.Flags().TimeVar(&startTime, "start-time", time.Time{}, []string{time.RFC3339}, "Time at which the silence will start to take effect (defaults to now)")
+	cmd.Flags().TimeVar(&endTime, "end-time", time.Time{}, []string{time.RFC3339}, "Time at which the silence will expire (default to one week from now)")
+	cmd.Flags().DurationVar(&duration, "expire-after", 0, "Duraration (e.g. 24h, 30m) after which the silence will expire - exclusive with --start-time & --end-time")
+	cmd.MarkFlagsMutuallyExclusive("start-time", "expire-after")
+	cmd.MarkFlagsMutuallyExclusive("end-time", "expire-after")
+
+	userName := "osdctl"
```

golangci-lint (2.12.2)

[error] ineffectual assignment to userName

(ineffassign)